### DISCUSSION TITLE
[1173400]Futa Fix Dick Dine and Dash

### DISCUSSION THREAD

**Discussion by Dragon-stars** [pjy612/SteamManifestCache#214] [Share]
appid:1173400
keys:
```
1173401;0cb6481a0def4668ab4eba9df8ee0d97c6f987427c64ea11aed7ebe0c44a919f
1173402;463c07eddf2dd982c46cad273257d96d3d77043b17b7d77e39c1d2a0a52380ef
1173400;af318612cc768512a746597271904259a8c12c54cf05092b0dcae852656d6a73
```
[密钥+清单.zip](https://github.com/pjy612/SteamManifestCache/files/13798417/%2B.zip)
内测码：41FB4B744818EB18
Email：2303039323@qq.com

**Comment by Dragon-stars** [comment]
@pjy612 您好，我想知道内测的成就导入使用怎么操作，内测版还有哪些功能

**Reply by pjy612** [reply]
看顶置的关于内测，成就入库游戏就行了。
如果发现游戏有成就，但是 入库时有清单却没成就，可以反馈。

**Reply by Dragon-stars** [reply]
![image](https://github.com/pjy612/SteamManifestCache/assets/68679568/0f4f199c-2873-4459-adf0-305e01a897b7)
您好，如图目前下的这款游戏就没显示成就系统，还有前两日玩的四海兄弟最终版（已经通关卸载）也没显示成就
@pjy612

**Reply by pjy612** [reply]
但凡你贴个id 都比你贴图快

**Reply by Dragon-stars** [reply]
您好，请问现在还有什么仓库/途径可以获取清单密钥吗，可以分享下吗？

**Reply by pjy612** [reply]
隔壁 BlankTming
